src/commands/DumpRulesCommand.ts

```diff
@@ -14,10 +14,62 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { Mjolnir } from "../Mjolnir";
 import { RichReply } from "matrix-bot-sdk";
+import { Mjolnir } from "../Mjolnir";
+import { RULE_ROOM, RULE_SERVER, RULE_USER } from "../models/BanList";
 import { htmlEscape } from "../utils";
 
+/**
+ * List all of the rules that match a given entity.
+ * The reaso why you want to test against all rules and not just e.g. user or server is because
+ * there are situations where rules of different types can ban other entities e.g. server ACL can cause users to be banned.
+ * @param roomId The room the command is from.
+ * @param event The event containing the command.
+ * @param mjolnir A mjolnir to fetch the watched lists from.
+ * @param entity e.g. a user or room id.
```

Comment:
Nit: Could you document whether it works with room aliases? Server names?

Reply:
The thing about room aliases is that Mjolnir already has a problem where if you add a room alias, it will not resolve the alias and create a `m.policy.rule.room` with the alias as the entity rather than the room id. Now if you go by principle of least astonishment, maybe it is, but it could have been made that way to mean that "This alias should be disallowed".

Reply:
But for now the command should be consistent with that and not resolve aliases either